### PR TITLE
styles(*) Fix broken & missing styles

### PR DESCRIPTION
**Pull request by travispamaral**
### Summary
- Updates type sizing classes (`.xl`-> `.type-xl`)
- Adds color classes to utils (`.color-blue-base`)
- Adds missing styles for `<select />`
- Fixes svg sizing/spacing in KButton as well as the slot checks

### PR Checklist
- [x] Does not introduce dependencies
- [x] **Functional:** all changes do not break existing APIs and if so, bump major version.
- [x] **Tests pass:** check the output of yarn test packages/<Kongponent>
- [x] **Naming:** the files and the method and prop variables use the same naming conventions as other Kongponents
- [x] **Framework style:** abides by the essential rules in Vue's style guide
- [x] **Cleanliness:** does not have formatting issues, unused code (e.g., console.logs), or leftover comments
- [x] **Docs:** includes a technically accurate README, uses JSDOC where appropriate

### CODE REVIEW

**Comment by xitongzou** [comment]
can you post a before/after screenshot just to see the visual changes easier? thanks

**Comment by travispamaral** [comment]
> can you post a before/after screenshot just to see the visual changes easier? thanks

You can view the changes on the docs preview. The only _real_ change is the KButton bug, the rest are additions for stuff we didn't have support for (css file)

File: packages/styles/forms/_inputs.scss [hunk 1]
```diff
@@ -1,7 +1,7 @@
 @import "labels";
 
 .k-input,
-.form-control {
+.form-control{
```
Comment (xitongzou) [inline packages/styles/forms/_inputs.scss:4]: dont think this change was intended

File: packages/styles/_typography.scss [hunk 2]
```diff
@@ -19,9 +19,17 @@
...
 }
 
-
+/*  Colors
+========================================================================== */
+// For each color in $colors variable
+// Create class (.color-blue-base)
+@each $name, $color in $colors {
+  .color-#{$name} {
+    color: var(--#{$name})
```
Comment (darrenjennings) [inline packages/styles/_typography.scss:33]: can you add a comment in utilities.md that `.color-{name}` is a utility class? Also would be nice to cross link in `colors.md`

```md
### Colors
| Class                   | Properties | Example
| :---------------------- |:-----------|:-----------
| .color-{color} |  color: var(--{color}); | `class="color-blue-base"`
```

File: packages/styles/_typography.scss [hunk 3]
```diff
@@ -3,9 +3,9 @@
 /* Regular (Sans) Font Sizes
 ========================================================================== */
 // For each size in the $type-sizes variable
-// Create class (.xxl)
+// Create class (.type-xxl)
 @each $name, $size in $type-sizes {
-  .#{$name} {
+  .type-#{$name} {
```
Comment (darrenjennings) [inline packages/styles/_typography.scss:8]: need to update `type.md`

Comment (darrenjennings) [inline packages/styles/_typography.scss:8]: this is a breaking change, do any kongponents use this class? Do we use it in kong-admin or khcp?

Comment (travispamaral) [reply packages/styles/_typography.scss:8]: no the only type references in either repo are in css and not class names

**Pull request merged**